docs/developer/example-configurations/gitlab.rst

```diff
@@ -1,93 +1,69 @@
-GitLab client application created in gitlab.com (Renku-local) into the designated places.
-
-Now start the Renku platform following the steps described in the `Renku charts README`_,
-replacing the final :code:`helm upgrade` command with the following:
-
-.. _`Renku charts README`: https://github.com/SwissDataScienceCenter/renku/blob/master/charts/README.rst
-
-.. code-block:: console
-
-  $ helm upgrade renku --install --namespace renku \
-    -f minikube-values.yaml -f example-configurations/minikube-values-gitlab.yaml \
-    --set global.renku.domain=$(minikube ip) \
-    --set ui.jupyterhubUrl=http://$(minikube ip)/jupyterhub \
-    --set ui.gatewayUrl=http://$(minikube ip)/api \
-    --set gateway.keycloakUrl=http://$(minikube ip) \
-    --set notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://$(minikube ip)/api/auth/jupyterhub/token \
-    --set notebooks.jupyterhub.auth.gitlab.callbackUrl=http://$(minikube ip)/jupyterhub/hub/oauth_callback \
-    ./renku
-
-Configure the identity provider
--------------------------------
-
-Open :code:`http://<your-minikube-ip>/auth` in your browser and login to the
-admin console using admin/admin. Under Identity Providers choose "GitLab" from
-the "Add Provider..." drop-down menu. Add the Application Id and Secret from
-the Renku-local application created in Step 1. Save the identity provider and
-logout from the Keycloak admin panel.
-
-Configure GitLab CI
--------------------
-
-For each created project you will have to modify the :code:`.gitlab-ci.yaml`
-in the repository for the image build to work. In the image_build job
-
-- remove the :code:`image-build` tag
-- add the following entry:
-
-.. code-block:: console
-
-  services:
-    - docker:dind
+- Depending on how runners are configured on the GitLab instance which you are using,
+  you might have to adapt the ``.gitlab-ci.yaml`` file in your projects. See the
+  :ref:`paragraph on configuring GitLab CI <runner_extra_config>`.
+
+
+gitlab.com
+----------
+The extra configuration needed for a deployment against ``gitlab.com`` is in a
+specific `values file <https://github.com/SwissDataScienceCenter/renku/blob/master/charts/example-configurations/external-gitlab-values.yaml>`_.
+This file can also be used as secondary values file in a Renku deployment which is
+running in the cloud.
+
+
+renkulab.io
+-----------
+You can also use the GitLab instance of the SDSCs public renku deployment
+`<renkulab.io/gitlab>`_ as GitLab backend. The same
+`values file <https://github.com/SwissDataScienceCenter/renku/blob/master/charts/example-configurations/external-gitlab-values.yaml>`_.
+has the necessary configurations for such a deployment indicated. All the steps
+necessary for such a setup are identical to the ones when using ``gitlab.com``,
+except for the configuration of ``renkulab.io/gitlab`` as upstream identity provider
+your Keycloak instance. Instead of ``gitlab.com`` pick the generic OICD provider
```

Comment:
is this missing some words here?

Reply:
clearly, yes.. `as the upstream identity provider FOR your Keycloak instance.` Will fix it.